appveyor.yml

```diff
@@ -14,9 +14,9 @@ cache:
 - build_msvc\cache
 init:
 - cmd: set PATH=C:\Python36-x64;C:\Python36-x64\Scripts;%PATH%
-- cmd: set BITCOINPATH=%APPVEYOR_BUILD_FOLDER%\build_msvc\%PLATFORM%\%CONFIGURATION%
-- cmd: set BITCOIND=%BITCOINPATH%\bitcoind.exe
-- cmd: set BITCOINCLI=%BITCOINPATH%\bitcoin-cli.exe
+- ps:  $env:BITCOINPATH = $env:APPVEYOR_BUILD_FOLDER + "\build_msvc\" + $env:PLATFORM + "\" + $env:CONFIGURATION
+- ps:  $env:BITCOIND = $env:BITCOINPATH + "\bitcoind.exe"
+- ps:  $env:BITCOINCLI = $env:BITCOINPATH + "\bitcoin-cli.exe"
```

Comment:
You can keep like this is you feel lazy, but 

` $env:BITCOINPATH = $env:APPVEYOR_BUILD_FOLDER\build_msvc\$env:PLATFORM\$env:CONFIGURATION` is more readable

Reply:
done